From 2e97957cdc037ba1ea5d0697818302f10205f30f Mon Sep 17 00:00:00 2001
From: Apollon Oikonomopoulos <apoikos@gmail.com>
Date: Sun, 29 Sep 2013 23:04:13 +0300
Subject: [PATCH 072/179] DOC: update manpage reference to haproxy-en.txt

The manpage refers to haproxy-en.txt, which is obsolete. Update the reference
to point to configuration.txt, together with the location on Debian systems.

Also capitalize "Debian".

Signed-off-by: Apollon Oikonomopoulos <apoikos@gmail.com>
---
 doc/haproxy.1 | 5 ++---
 1 file changed, 2 insertions(+), 3 deletions(-)

diff --git a/doc/haproxy.1 b/doc/haproxy.1
index 6296767..3254fe6 100644
--- a/doc/haproxy.1
+++ b/doc/haproxy.1
@@ -198,9 +198,8 @@ This signal is intercepted and ignored on systems without \fBMSG_NOSIGNAL\fP.
 
 .SH SEE ALSO
 
-A much better documentation can be found in haproxy-en.txt. On debian
-systems, you can find this file in
-/usr/share/doc/haproxy/haproxy-en.txt.gz.
+A much better documentation can be found in configuration.txt. On Debian
+systems, you can find this file in /usr/share/doc/haproxy/configuration.txt.gz.
 
 .SH AUTHOR
 
-- 
1.8.1.5

